lint_test.go

```diff
@@ -7,5 +7,5 @@ import (
 )
 
 func TestAll(t *testing.T) {
-	testutil.TestAll(t, Funcs)
+	testutil.TestAll(t, Funcs, "")
```

Comment:
Thanks, I'll leave this with @dominikh to advise on the preferred solution.

Note, this mimics the behaviour in go-staticcheck: https://github.com/dominikh/go-staticcheck/commit/02440e0192ce5dfbfe6725b233028bd6e68df576#diff-e646beadeeb05d6a7afc367e1850240dR10